gradle/libs.versions.toml

```diff
@@ -35,6 +37,11 @@ swagger-codegen-plugin = { module = "gradle.plugin.org.hidetake:gradle-swagger-g
 # we need to manually upgrade here in order to have the library compatible with jackson
 swagger-jersey2-jaxrs = { module = "io.swagger:swagger-jersey2-jaxrs", version = { strictly = "1.6.2" } }
 
+
+springdoc-openapi-plugin = { module = "org.springdoc:springdoc-openapi-gradle-plugin", version = "1.6.0" }
+json2yaml-plugin = { module = "com.ascert.open:json2yaml", version = "1.1.1" }
```

Comment:
that seems unrequired as we're using kotlinx-serialization to do the conversion in `Json2YamlTask`

Reply:
thanks for spotting that, was in fact left from a task implementation using that plugin